docs/guides/avs-operator-guide.mdx

```diff
@@ -0,0 +1,38 @@
+# Hyperlane AVS Operator Guide for Holesky
+
+Hyperlane validators are extremely lightweight off-chain agents responsible for observing the [Mailbox](../protocol/mailbox.mdx) on the source chain contract and, if required, signing a merkle root to faciliate message transmission to remote chains.
+
+### Prerequisites
+
+- Install the `@hyperlane-xyz/cli` package versioned `3.12.2` or later. Follow instructions for installation [here](https://docs.hyperlane.xyz/docs/reference/cli).
+
+## Register your operator
+
+:::warning
+
+There is a distinction between the operator key and the AVS signing key. The operator key is meant for just interacting with the AVS contracts whereas the AVS signing key is used for active operations, which in our case will be signing checkpoints with a Hyperlane Validator. As of May 15th 2024, EigenLayer doesn't support attesting to an AVS signing key while registering as an operator but that will be changing in short order. You can read more details about the keys [here](https://docs.eigenlayer.xyz/eigenlayer/operator-guides/key-management/intro#operator-keys).
+
+:::
+
+To register with Hyperlane, operators must first register as an operator on the Eigenlayer smart contracts on Holesky testnet. You can follow Eigenlayer's instructions [here](https://docs.eigenlayer.xyz/eigenlayer/operator-guides/operator-installation#operator-configuration-and-registration) (only need the ECDSA key).
+
+Next, to register your operator to Hyperlane AVS, you can register your operator key (generated via EigenLayer CLI) by creating an ECDSA signature and submitting it via the Hyperlane CLI:
+
+```bash
+ hyperlane avs register --operatorKeyPath <PATH_TO_OPERATOR_KEY> --chain holesky --key <HYP_KEY>
+```
+
+### Running the validator
+
+Follow the validator guide [here](../operate/validators/run-validators.mdx) with the AWS setup to run the validator.
+
+You can monitor Eigenlayer specific endpoints via the `--metrics` port you specified in the validator configuration. Read more about the example endpoints and what should be expected [here](https://docs.eigenlayer.xyz/eigenlayer/avs-guides/spec/api/).
+
+The supported endpoints are:
+
+| Endpoint                                  | Description                                |
+| ----------------------------------------- | ------------------------------------------ |
+| **GET /eigen/node**                       | for node info                              |
+| **GET /eigen/node/health**                | for node health                            |
+| **GET /eigen/node/services**              | for a list of services the node is running |
+| **GET /eigen/node/`{service_id}`/health** | for health of a specific service           |
```

Comment:
What is `service_id` here? IIRC, we didn't really have subservices, so maybe lets just omit this information?

Reply:
We have serviceId `hyperlane-validator-indexer` and `hyperlane-validator-submitter` hardcoded. So, I'll remove it until it's not hardcoded.